src/RefractDataStructure.cc

```diff
@@ -158,7 +158,7 @@ namespace drafter {
         //std::transform(elements.begin(), elements.end(),
         //               std::back_inserter(result),
         //               std::bind2nd(std::ptr_fun(MsonElementToRefract), nestedTypeName));
-
+        
```

Comment:
@obihann Would you be able to remove all the trailing white-space you've added please.